helm-charts/interoperator/templates/operator_api_app.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: {{ .Release.Name }}-op-apis-service
+  name: {{ .Release.Name }}-op-apis-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.operator_apis.service.type }}
+  ports:
+  - name: http
+    port: 80
+    targetPort: op-apis-port
+  - name: https
+    port: 443
+    targetPort: op-apis-port
+  selector:
+    app: {{ .Release.Name }}-op-apis-app
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    service: {{ .Release.Name }}-op-apis-service
+    {{- if .Values.cluster.ssl }}
+    garden.sapcloud.io/purpose: managed-cert
+    {{- end }}
+  name: {{ .Release.Name }}-op-apis-service-ingress
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+  - host: {{ .Values.cluster.op_apis_host }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .Release.Name }}-op-apis-service
+          servicePort: 80
+  {{- if .Values.cluster.ssl }} 
+  tls:
+  - hosts:
+    - {{ .Values.cluster.op_apis_host }}
+    secretName: {{ .Values.cluster.sslsecret }}
+  {{- end }} 
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-op-apis-app
+  labels:
+    app: {{ .Release.Name }}-op-apis-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-op-apis-app
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-op-apis-app
+      annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 6 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.operator_apis.image.repository }}:{{ .Values.operator_apis.image.tag }}"
```

Comment:
I did not see the values.yml. Is it missed to be added ?

Reply:
yes, it was missed as I had some unstaged changes for testing.. I will push it along with above suggested changes.

Reply:
Added `values.yml`